### PR TITLE
fix(core/biblio): normalize finding references

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: src/core/biblio.js [hunk 1]
```diff
@@ -198,22 +195,27 @@ function bibref(conf) {
...
-        const msg = `Circular reference in biblio DB between [\`${ref}\`] and [\`${key}\`].`;
-        pub("error", msg);
-      } else {
-        key = refcontent.aliasOf;
-        refcontent = conf.biblio[key];
-        circular.add(key);
+  function makeBibKeyFinder() {
+    const keys = Object.keys(conf.biblio);
+    return function getRefContent(ref) {
+      let refcontent = keys.find(
```
Comment (sidvishnoi) [inline src/core/biblio.js]: this should be something like:
``` js
      let key = keys.find(
        key => key.toLocaleLowerCase() === ref.toLocaleLowerCase()
      );
      let refcontent = conf.biblio[key];
```

Comment (marcoscaceres) [reply src/core/biblio.js]: good point

**Comment by sidvishnoi** [comment]
This was the reason for references not being found:

https://github.com/w3c/respec/blob/b2a393aa1edc61b029e224b63a64eaf8f6b6e035/src/core/biblio.js#L202

https://github.com/w3c/respec/blob/b2a393aa1edc61b029e224b63a64eaf8f6b6e035/src/core/biblio.js#L253

**Comment by sidvishnoi** [comment]
closing via https://github.com/w3c/respec/pull/1703

**Pull request closed without merging**